pkg/odo/cli/dev/dev.go

```diff
@@ -169,10 +170,17 @@ func (o *DevOptions) Run() error {
 		Namespace: o.Context.GetProject(),
 	}
 	var path = filepath.Dir(o.Context.EnvSpecificInfo.GetDevfilePath())
+	devfileName := o.EnvSpecificInfo.GetDevfileObj().GetMetadataName()
+	namespace := o.GetProject()
 
-	fmt.Fprintf(o.out, "Starting your application on cluster in developer mode\n")
+	// Output what the command is doing / information
+	log.Title("Developing the Devfile: "+devfileName,
```

Comment:
This might be a little missleading. The user is not developing a Devfile, but an application. 

Maybe `Developing using the "+devfileName+" Devfile`?